tests/helper.h

```diff
@@ -45,5 +53,9 @@ void package_files_setup(PackageFilesFixture *fixture,
                          gconstpointer user_data);
 void package_files_teardown(PackageFilesFixture *fixture,
                             gconstpointer user_data);
+void file_copy_setup(FileCopyFixture *fixture,
+                     gconstpointer filename);
+void file_copy_teardown(FileCopyFixture *fixture,
+                        gconstpointer user_data);
```

Comment:
How about keeping this consistent with the other fixtures and rename it to `copy_file_*` and `CopyFileFixture`?